### ISSUE TITLE
Source for citation numbers

### ISSUE THREAD

**Issue by bfirsh** [terryum/awesome-deep-learning-papers#23]
Out of interest, where did you get the numbers for citations? Did you use any mechanism for discovering these papers, or was it purely your own knowledge?

**Comment by terryum** [comment]
Not really. I hope I could find a way to automatically update the list of good papers, but there's not. I just searched papers and tracked their references. 
For example, if I find a famous paper, I check the author's google citation page to see if he/she has more good papers. Also, I check the papers that cite the famous paper, also the papers that is cited by the famous papers. It's kinda infinite chains...haha.

**Comment by terrycafe520** [comment]
You have searched papers and tracked their references , and if your have the data such as H index ?  I can write some code to show the data use chartjs tools.

**Comment by bthayer2365** [comment]
Seems like you're doing exactly what this algorithm does. I'll look into implementations tomorrow. https://en.m.wikipedia.org/wiki/HITS_algorithm

**Comment by aleksas** [comment]
You can use google scholar data. There is a project on GitHub to retrieve publication metadata - https://github.com/ckreibich/scholar.py/blob/master/scholar.py. 

Example of running script for a paper from the list:

`python run.py -c 1 -t -A "Training very deep networks" --after=2015 --before=2017`
